jobs/ci-run/integration/gen/test-smoke.yml

```diff
@@ -95,7 +95,7 @@
 
 - job:
     name: test-smoke-test-build-aws
-    node: ephemeral-focal-small-amd64
+    node: ephemeral-focal-8c-32g-amd64
```

Comment:
ditto

Reply:
Had to be bumped up for the moment because they were being OOM killed.